src/vl-tooltip.js

```diff
@@ -352,84 +376,76 @@
   }
 
   /**
-  * Get one field value from a datum.
-  * @param {Object} datum - the datum of an item.
+  * Get a field value from a data map.
+  * @param {d3.map} itemData - a map of item.datum
   * @param {string} field - the name of the field. It can contain "." to specify
-  * that the field is not a direct child of datum.
-  * @return the field value, or undefined if the value cannot be found.
+  * that the field is not a direct child of item.datum
+  * @return the field value on success, undefined otherwise
   */
   // TODO(zening): Mute "Cannot find field" warnings for composite vis (issue #39)
-  function getValue(datum, field) {
-    if (field.includes(".")) {
-      var accessors = field.split(".");
-      var value = datum;
-      var path = "";
+  function getValue(itemData, field) {
+    var value;
+    
+    var accessors = field.split('.');
```

Comment:
This part is interesting.  I wonder if `field:`x.a`` works in Vega-Lite.  

Even if it works, I wonder how this works with `{aggregate: 'mean', field: 'x.a'}`.  (Would it break things?)